ruby/vampires.rb

```diff
@@ -0,0 +1,63 @@
+puts "How many employees are there ?"
+employees=gets.chomp.to_i
+for i in 1..employees do
+  puts "What is your name?"
+  name = gets.chomp
+  puts "How old are you?"
+  age = gets.chomp.to_i
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+  puts "Our company cafeteria serves garlic bread. Should we order some for you?(y/n)"
+  garlic = gets.chomp
+  puts "Would you like to enroll in the company's health insurance?(y/n)"
+  insurance = gets.chomp
+
+  # p name, age, birth_year, garlic, insurance
+
+  probably_not_a_vampire=false;
+  #
+  # in hind sight probably should have broken age check
+  # logic out and added another boolean for age_check
+  #
+  if ((2017-birth_year-age).abs<=1) && ((garlic=="y") || (insurance=="y"))
+    probably_not_a_vampire=true;
+  end
+  probably_a_vampire=false
+  if ((2017-birth_year-age).abs>1) && ((garlic=="n") || (insurance=="n"))
+    probably_a_vampire=true
+  end
+  almost_certainly_a_vampire=false
+  if (((2017-birth_year-age).abs>1) && (garlic=="n") && (insurance=="n"))
+    almost_certainly_a_vampire=true
+  end
+  definitely_a_vampire=false
+  if name=="Drake Cula" || name=="Tu Fang"
+    definitely_a_vampire=true
+  end
+
+  allergies=""
+  while allergies!="done"
+    puts "Please list any allergies. Use \"done\" when finished."
+    allergies=gets.chomp
+    if allergies=="sunshine"    
+      probably_a_vampire=true
+      break
+    end
+  end
+  #
+  # a case statement would seem to be more readable code wise
+  # v.s. a stack of if/elseif/else but they might be interchangeable
+  case 
+  when definitely_a_vampire

```

Comment:
If you refactored the conditionals to just update a results variable with the strings, then you can remove this case statement.